### PR TITLE
fix: health.proto has name conflict with the one in google.golang.org/grpc

### LINKED ISSUE #2394: grpc/health/v1/health.proto has a name conflict over grpc.health.v1.HealthCheckResponse
### Environment

- Server: Dubbo-go, v3.1.0
- Client: Dubbo-go, v3.1.0
- Protocol: Dubbo
- Registry: Nacos

### Issue description

health check in "dubbo.apache.org/dubbo-go/v3/protocol/dubbo3/health/triple_health_v1" conflicts with the one in "google.golang.org/grpc/health/grpc_health_v1".

Code to show the bug:
```
package main

import (
	_ "dubbo.apache.org/dubbo-go/v3/imports"
	_ "google.golang.org/grpc/health/grpc_health_v1"
)

func main() {

}

```
> To execute the code above, it's necessary to create a `go.mod` file and update the dependencies.

### Logs

<details><summary>Click me to check logs</summary>
After running code above， panic occurs：

```
panic: proto: file "grpc/health/v1/health.proto" has a name conflict over grpc.health.v1.HealthCheckResponse
	previously from: "dubbo.apache.org/dubbo-go/v3/protocol/dubbo3/health/triple_health_v1"
	currently from:  "google.golang.org/grpc/health/grpc_health_v1"
See https://protobuf.dev/reference/go/faq#namespace-conflict
```

</details>

> See doc in: https://protobuf.dev/reference/go/faq#namespace-conflict. 
> This issue arises due to the `protocol/dubbo3/health/triple_health_v1/health.proto` having an identical package name as `https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto`.

### PR DESCRIPTION
**Pull request by HubQin**
health check in "dubbo.apache.org/dubbo-go/v3/protocol/dubbo3/health/triple_health_v1" conflicts with the one in "google.golang.org/grpc/health/grpc_health_v1".

Fixes #2394

### CODE REVIEW

File: protocol/dubbo3/health/triple_health_v1/health.pb.go [hunk 1]
```diff
@@ -80,11 +75,11 @@ func (x HealthCheckResponse_ServingStatus) String() string {
 }
 
 func (HealthCheckResponse_ServingStatus) Descriptor() protoreflect.EnumDescriptor {
-	return file_protocol_dubbo3_health_triple_health_v1_health_proto_enumTypes[0].Descriptor()
+	return file_health_proto_enumTypes[0].Descriptor()
```
Comment (justxuewei) [inline protocol/dubbo3/health/triple_health_v1/health.pb.go:78]: Why is this function changed?

Comment (justxuewei) [reply protocol/dubbo3/health/triple_health_v1/health.pb.go:78]: Is your triple protoc right?

**Comment by justxuewei** [comment]
Btw, you need to add a Apache license for those files.

File: protocol/dubbo3/health/triple_health_v1/health.pb.go (hunk 1 above)
Comment (HubQin) [reply protocol/dubbo3/health/triple_health_v1/health.pb.go:78]: yes，it is auto-gen by protoc. command to generate:
```
protoc --go_out=. --go_opt paths=source_relative --go-triple_out=. ./health.proto --go-triple_opt paths=source_relative
```

**Comment by HubQin** [comment]
> Btw, you need to add an Apache license for those files.

@justxuewei updated

**Review by ev1lQuark — APPROVED** [review]
LGTM

**Review by justxuewei — APPROVED** [review]
Lgtm, thanks!

**Pull request merged**